iati/iati/settings/base.py

```diff
@@ -61,6 +61,8 @@
 
     'wagtail.core.middleware.SiteMiddleware',
     'wagtail.contrib.redirects.middleware.RedirectMiddleware',
+    # For determining browser locale
+    'django.middleware.locale.LocaleMiddleware',
```

Comment:
I believe this needs to be placed further up the list: https://docs.djangoproject.com/en/2.0/topics/i18n/translation/#how-django-discovers-language-preference